scratch_kotlin_lib/src/main/java/dev/sgrem/scratch_kotlin_lib/Week4Homework.kt

```diff
@@ -0,0 +1,160 @@
+package dev.sgrem.scratch_kotlin_lib
+
+fun main() {
+
+    // 1. Task Data:
+    data class Task(
+        val title: String,
+        // var priority: Int = 1,
+        var priority: Priority = Priority.HIGH,
+        var completed: Boolean = false
+    )
+
+    // 2. Task List
+    val taskList: MutableList<Task> = mutableListOf()
+
+    // 3. Add Task:
+    fun addTask(taskList: MutableList<Task>,
+                title: String,
+                // priority: Int = 1,
+                priority: Priority = Priority.HIGH,
+                completed: Boolean = false
+                ){
+        taskList.add(Task(title, priority, completed))
+    }
```

Comment:
Minor, but indentation here seems a bit off.  Not sure if this was Android Studio that did it but typically indentation is 4 spaces, even for multi-line definitions of functions, and there's usually a space between the ending parenthesis and the starting curly brace:

```suggestion
    fun addTask(taskList: MutableList<Task>,
            title: String,
            // priority: Int = 1,
            priority: Priority = Priority.HIGH,
            completed: Boolean = false
    ) {
        taskList.add(Task(title, priority, completed))
    }
```